### DISCUSSION TITLE
Integrating with AWS Microsoft AD

### DISCUSSION THREAD

**Discussion by lkwjohn** [gravitational/teleport#10896] [Q&A] [answered]
I'm evaluating for my team to adore teleport, so I'm wondering if teleport support the integration to AWS Microsoft AD? 
Also if possible, following the guide on ADFS is sufficient?

**Comment by Valien** [comment, marked as answer]
Hi there! I believe AWS MS AD is supported. I think a guide is being written for it but it might not be out yet. You can try the ADFS guides and report back here or ping us in our community slack channel -- https://goteleport.slack.com/
